doc/ssl/DTLS_get_data_mtu.pod

```diff
@@ -0,0 +1,36 @@
+=pod
```

Comment:
The location for this file needs to change now that we have restructured the doc directory